tests/foreman/api/test_permission.py

```diff
@@ -34,6 +35,14 @@ def setUpClass(cls):
             cls.permissions[None].remove('view_log_viewer')
             cls.permissions[None].remove('view_search')
 
+        result = ssh.command('rpm -qa | grep rubygem-foreman_openscap')
+        if result.return_code != 0:
+            cls.permissions.pop('ForemanOpenscap::Policy')
+            cls.permissions.pop('ForemanOpenscap::ScapContent')
+            cls.permissions[None].remove('destroy_arf_reports')
```

Comment:
Neat-o. It helped me to understand it better when I realized that `permissions[None]` contained a `list` :)

```python
In [6]: foo = {'a': 1, 'b': 2, None: ['c', 'd', 'e']}

In [7]: foo[None].remove('c')

In [8]: foo
Out[8]: {None: ['d', 'e'], 'a': 1, 'b': 2}
```